animation_nodes/data_structures/meshes/mesh_data.pxd

```diff
@@ -10,4 +10,4 @@ cdef class Mesh:
         dict derivedMeshDataCache
         object uvMaps
         object vertexColorLayers
```

Comment:
Those should be removed. The same goes for `materialIndices`.

Reply:
Removed.